packages/autoloader/src/functions.php

```diff
@@ -130,10 +129,9 @@ function reset_maps_after_update( $response, $hook_extra, $result ) {
 		$plugin_dir  = str_replace( '\\', '/', WP_PLUGIN_DIR );
 		$plugin_path = trailingslashit( $plugin_dir ) . trailingslashit( explode( '/', $plugin )[0] );
 
-		if ( is_readable( $plugin_path . 'vendor/autoload_functions.php' ) ) {
-			// The plugin has a v2.x autoloader, so reset it.
-			$jetpack_autoloader_latest_version = null;
```

Comment:
Since it was not obvious to me, I'm going to mark this line as the main fix. This is the way we're stopping the autoloader from searching for the newest version and just using the one it has already found in `set_up_autoloader`, correct?

Reply:
There are two issues that need to be solved:

- The autoloader needs to stop trying to find latest installed autoloader during an plugin update. 
- We need to stop the autoloaders that are already released from trying to find the latest installed autoloader during a plugin update.

### Stop trying to find latest installed autoloader during an plugin update. 

Currently, when a plugin with a v2.x autoloader updates, `rest_maps_after_update()` is called. In that function, we reset both the `$jetpack_autoloader_latest_version` and `$jetpack_packages_classmap` globals. Then we call `set-up_autoloader()`.

In `set_up_autoloader()`, we [check the value](https://github.com/Automattic/jetpack/blob/master/packages/autoloader/src/functions.php#L83) of `$jetpack_autoloader_latest_version` to determine if we need to find the latest autoloader version. Since we set that variable to null in `reset_maps_after_update()`, the `find_latest_autoloader()` method is called.

In this change, I propose that we no longer reset the value of `$jetpack_autoloader_latest_version` in `reset_maps_after_update()`. Then, `find_latest_autoloader()` will not be called. This should prevent the error that could occur during an update.